src/BehaviorsSDKManaged/Microsoft.Xaml.Interactivity.WinUI/Microsoft.Xaml.Interactivity.WinUI.csproj

```diff
@@ -170,28 +164,15 @@
       <Link>Resources\zh-TW\Strings.resw</Link>
     </PRIResource>
   </ItemGroup>
-  <Import Project="..\Microsoft.Xaml.Interactivity.Shared\Microsoft.Xaml.Interactivity.Shared.projitems" Label="Shared" />
   <PropertyGroup Condition=" '$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &lt; '14.0' ">
     <VisualStudioVersion>14.0</VisualStudioVersion>
   </PropertyGroup>
+  <Import Project="..\Microsoft.Xaml.Interactivity.Shared\Microsoft.Xaml.Interactivity.Shared.projitems" Label="Shared" />
   <Import Project="..\..\..\scripts\Microsoft.Xaml.Behaviors.Signing.targets" />
-  <PropertyGroup>
-    <PackageId>Microsoft.Xaml.Behaviors.WinUI.Managed</PackageId>
-    <Title>Microsoft.Xaml.Behaviors.WinUI.Managed</Title>
-    <Authors>Microsoft</Authors>
-    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
```

Comment:
Are these also shared elsewhere now?

Reply:
Accidentally removed with the whole rename, fixed! Thank you 🙂